lib/findAllFiles.js

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+
+import glob from 'glob';
+
+import lastUpdate from './lastUpdate';
+
+export default function findAllFiles(workingDirectory, excludes) {
+  return new Promise((resolve, reject) => {
+    glob('./**/*.js*', {
```

Comment:
I wonder if there is a more specific glob pattern that we could use. Like, does this work? 
```
.*/**/*.@(js|jsx|json)
```

https://github.com/isaacs/node-glob#glob-primer